tests/batch.test.ts

```diff
@@ -33,11 +33,14 @@ describe.each([{ permission: "Master" }, { permission: "Admin" }])(
       const batches = await client.batches.getBatches();
       const batch = await client.batches.getBatch(batches.results[0].uid);
       expect(batch.uid).toEqual(batches.results[0].uid);
+
+      // Can't use toMatchSnapshot because the output changes every time
```

Comment:
I'm not sure I understand this comment, can you explain to me what the issue is?

Reply:
The API response contains data that can change, like batch processing information. So the snapshots usually don't match from one run to another.